src/ripple/app/paths/Pathfinder.cpp

```diff
@@ -1185,8 +1185,10 @@ void fillPaths (Pathfinder::PaymentType type, PathCostList const& costs)
 void Pathfinder::initPathTable ()
 {
     // CAUTION: Do not include rules that build default paths
-    fillPaths(
-        pt_XRP_to_XRP, {});
+
+    // When the app is destroyed and a new app is created, start with a fresh mPathTable
+    mPathTable.clear();
+    fillPaths (pt_XRP_to_XRP, {});
```

Comment:
I'm OK taking it out. If you do take out that line of code you'll quick hit an assert later in the code and it's not like the comment explains some extra-tricky reason the line needs to be there. I'll take it out.